GoogleDataTransportCCTSupport/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v1.2.0
+- Updates GDT dependency to improve backgrounding logic.
+- Reduces requests for background task creation. (#3893)
+
```

Comment:
Fix unbalanced background task creation in GDTCCTUploader (#3838)

Reply:
Done.